dev_guide/deployments.adoc

```diff
@@ -155,37 +162,38 @@ console] shows deployments in the *Browse* tab.
 
 == Canceling a Deployment
 
-To cancel a running or stuck deployment:
+To cancel a running or stuck deployment process:
 
 ----
 $ oc deploy --cancel dc/<name>
 ----
 
 [WARNING]
 ====
-The cancellation is a best-effort operation, and may take some time to complete.
-The replication controller may partially or totally complete deployment before
-the cancellation is effective. When canceled, the deployment automatically rolls
-back.
+The cancelation is a best-effort operation, and may take some time to complete.
+The replication controller may partially or totally complete its deployment before
+the cancelation is effective. When canceled, the deployment config will be automatically
+rolled back by scaling up the previous running replication controller.
 ====
 
 [[retrying-a-deployment]]
 
 == Retrying a Deployment
 
-To retry the last failed deployment:
+If the current revision of your deployment configuration failed to deploy, you can
```

Comment:
maybe worth to describe how to debug failures? (maybe worth to add a section dedicated for that)